controllers/user_controller.go

```diff
@@ -0,0 +1,60 @@
+package controllers
+
+import (
+	"net/http"
+
+	"github.com/drift-org/backend/models"
+	"github.com/gin-gonic/gin"
+	"github.com/kamva/mgm/v3"
+)
+
+type UserController interface {
+	Info(context *gin.Context)
+}
+
+type userController struct {
+}
+
+func NewUserController() UserController {
+	return &userController{}
+}
+
+func (ctrl *userController) Info(context *gin.Context) {
```

Comment:
Now that the route has changed, change the controller to match (so just `Get` is fine)